js/src/features/human-resouce/components/HumanResourceCalendar/HumanResourceCalendar.scss

```diff
@@ -1,54 +1,166 @@
-.react-calendar-timeline .rct-header-root {
-  background-color: #00a896;
-}
-
-.holiday {
-  background-color: rgba(255, 0, 0, 0.05);
-}
-
 .timetable {
+  position: relative;
+  overflow: hidden;
+  z-index: 1;
+
   .timetable-title {
     font-weight: bold;
+
+    &__month {
+      position: relative;
+      z-index: 10000;
+    }
   }
+
   .timetable-date {
-    font-size: 10;
+    font-size: 10px;
     font-weight: 200;
+    text-align: center;
+    position: relative;
+
+    &::before {
+      content: "";
+      position: absolute;
+      z-index: 1000;
+      height: 100%;
+      left: 1px;
+      bottom: calc(100% - 1px);
+      box-shadow: 0 0 0 1.6px #f0f2f5;
+    }
+
+    &::after {
+      content: "";
+      position: absolute;
+      z-index: 1000;
+      height: 100%;
+      left: calc(100% - 0.5px);
+      bottom: calc(100% - 1px);
+      box-shadow: 0 0 0 1.6px #f0f2f5;
+    }
   }
+
   .timetable-user {
     width: 150px;
     height: 20px;
-    padding: 0;
+    padding: 0 5px;
+    border: 2px solid grey;
 
     a {
       color: black;
       white-space: nowrap;
     }
   }
+
   .timetable-cell {
-    width: 20px;
-    border: 2px solid white;
-    background-color: rgb(142, 195, 81);
+    width: 25px;
+    height: 25px;
+    border: 2px solid grey;
+    position: relative;
 
     &.holiday {
-      background-color: violet;
+      background-color: #fadb14;
     }
+
     &.vacation {
-      background-color: skyblue;
+      background-color: rgba(142, 195, 81, 1);
+
+      &.not-approved {
+        background-color: rgba(142, 195, 81, 0.5);
+      }
     }
+
     &.sick {
-      background-color: slateblue;
+      background-color: rgba(255, 255, 255, 1);
+      color: darkred;
+      text-align: center;
+
+      &.not-approved {
+        background-color: rgba(255, 255, 255, 0.1);
+      }
     }
+
     &.dayon {
-      background-color: purple;
+      background-color: rgba(106, 90, 205, 1);
+
+      &.not-approved {
+        background-color: rgba(106, 90, 205, 0.5);
+      }
     }
+
     &.personal {
-      background-color: salmon;
+      background-color: rgba(250, 128, 114, 1);
+
+      &.not-approved {
+        background-color: rgba(250, 128, 114, 0.5);
+      }
     }
+
     &.weekend {
-      background-color: turquoise;
+      background-color: rgba(245, 34, 45, 1);
+    }
+  }
+
+  .timetable-info {
+    min-width: 25px;
+    height: 25px;
+    border: 2px solid grey;
+    text-align: center;
+  }
+
+  .select-row:hover {
+    position: relative;
+    th::before {
+      content: "";
+      width: 10000px;
+      height: 0;
+      position: absolute;
+      left: 0;
+      z-index: 4;
+      box-shadow: 0 -1px 0 1.5px black;
+    }
+
+    th::after {
+      content: "";
+      width: 10000px;
+      height: 0;
+      position: absolute;
+      left: 0;
+      z-index: 4;
+      box-shadow: 0 1px 0 1.5px black;
     }
-    &.not-approved {
-      opacity: 0.5;
+  }
+
+  .timetable-cell {
+    &:hover::before {
```

Comment:
![image](https://user-images.githubusercontent.com/9903441/83431912-c5b42080-a440-11ea-984c-42a778de5431.png)
I think we can fix it

Reply:
Ok seems like it's only on macs. I resolve it for now.